packages/app/src/views/Home.vue

```diff
@@ -1,13 +1,15 @@
 <template>
   <div class="home w-full flex flex-col">
-    <header class="flex py-5 justify-center text-4xl">
+    <header class="flex p-5 text-4xl">
       <Button
         :to="{ name: 'add-tag' }"
         icon="fas fa-plus-square"
         text="Tag hinzufügen"
         :text-size="5"
-        class="py-2 mx-4 flex-grow"
+        both-center
+        class="mx-4 flex-grow"
       />
+      <Button :to="{ name: 'settings' }" icon="fas fa-cog" class="py-2 ml-2 mr-4 px-2" />
```

Comment:
```suggestion
      <Button :to="{ name: 'settings' }" icon="fas fa-cog" class="py-2 mx-2 px-2" />
```

Reply:
okay,
fixed in 18e0a57672f8c08cc87053a75e68749a5cafd88e